oci/oci.go

```diff
@@ -0,0 +1,60 @@
+package oci
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/mrunalp/ocid/utils"
+)
+
+// New creates a new Runtime with options provided
+func New(name string, runtimePath string, sandboxDir string) (*Runtime, error) {
```

Comment:
Yeah, could take it out for now and add a separate parameter when there is a use case.